apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -805,11 +814,17 @@ impl Webserver {
         );
 
         if !project.is_production {
+            let path_prefix = project
+                .http_server_config
+                .path_prefix
+                .as_deref()
+                .map(|p| format!("/{}", p.trim_matches('/')))
+                .unwrap_or_default();
```

Comment:
a bit weird that it's `.unwrap_or_default()` here `.unwrap_or("/".to_string())` several lines down
we can probably use the same variable.

Reply:
Updated, lmk if there's anything else.